ocs_ci/ocs/constants.py

```diff
@@ -1484,15 +1484,15 @@
 }
 
 PERFORMANCE_PROFILE_REQUEST_CPU_VALUES = {
-    "mgr": "1500m",
-    "mon": "1500m",
+    "mgr": "1500m'",
+    "mon": "1500",
```

Comment:
due to what is the requirement we've changed PERFORMANCE_PROFILE_REQUEST_CPU_VALUES ? 

imho, this should be separated to different PR

Reply:
Addressed